astropy/time/tests/test_ut1.py

```diff
@@ -93,6 +93,16 @@ def test_ut1_to_utc(self):
             t_back = t.utc.ut1
             assert allclose_jd(t.jd, t_back.jd)
 
+    def test_empty_ut1(self):
+        """Converting a zero-length Time object from UTC to UT1
+        when an empty array is passed"""
+        from astropy import units as u
+        with iers_conf.set_temp('auto_download', False):
+            t = Time(['2012-06-30 12:00:00']) + np.arange(24) * u.hour
+            t_empty = t[[]].ut1
+            t_comp_empty = []
```

Comment:
The `allclose_jd` is really meant for comparing actual JD floats... Instead, I'd test a bit more explicitly:
```
assert isinstance(t_empty, Time)
assert t_empty.scale == 'ut1'
assert t_empty.size == 0
```